compile.sh

```diff
@@ -116,7 +116,7 @@ generate_test_row()
         # Reset color
         echo "{set:cellbgcolor!}" >> "$TMP_ADOC_FILE"
     fi
-    if $(xmlstarlet -q sel -t  -v "////testsuites/testsuite[$i]/testcase[$j]/failure" "$xml")
```

Comment:
The commit message is not accurate.
not using $() tests the return value of the command. (If it's 0, it's considered true)

Reply:
I wrote the explanation given by shellcheck.
https://www.shellcheck.net/wiki/SC2091